### PR TITLE
remove dotenv

### PR DESCRIPTION
**Pull request by maxbeatty**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If removing a declaration:
- [x] Delete the package's directory.
- [x] Add it to `notNeededPackages.json`.

### CODE REVIEW

File: types/oracledb/package.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+    "private": true,
+    "dependencies": {
+        "dotenv": "^8.2.0"
+    }
+}
```
Comment (ffflorian) [inline types/oracledb/package.json]: I think this package doesn't need `dotenv`.

**Review by ffflorian — CHANGES_REQUESTED** [review]

File: types/oracledb/package.json (hunk 1 above)
Comment (maxbeatty) [reply types/oracledb/package.json]: I'm not familiar with the package or its tests, but [dotenv is being used](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/6ec670c7d0220af69ba7dd67b386e3d23e00874b/types/oracledb/oracledb-tests.ts#L2)

Comment (ffflorian) [reply types/oracledb/package.json]: Whoops, you are right. My bad!

**Review by ffflorian — APPROVED** [review]

**Comment by ffflorian** [comment]
Alright, let's wait for https://github.com/microsoft/types-publisher/pull/666 :)

**Comment by maxbeatty** [comment]
Can someone help get this merged? I'm beyond my limits. [dotenv is requiring TypeScript 3.0](https://github.com/motdotla/dotenv/blob/70425a9c88e5fe5c3bd128fa973701279a76a9e3/types/index.d.ts#L1) because there were indiscernible errors in 2.x when running dtslint. Happy to help fix up motdotla/dotenv to make things easier here.

**Comment by ffflorian** [comment]
@maxbeatty You need to add `// TypeScript Version: 2.2` under the header in `types/dotenv-safe/index.d.ts`. If you add me to the contributors of your DefinitelyTyped fork I can also change it for you and push into your branch.

**Comment by ffflorian** [comment]
@maxbeatty I force-pushed into the branch to remove the failed rebase. :slightly_smiling_face:

**Review by ffflorian — APPROVED** [review]

**Comment by ffflorian** [comment]
Ping @jussikinnula @borekb @enaeseth

**Pull request merged**